### PR TITLE
Added some badges in README file

### LINKED ISSUE #39: add some badges to the README
### **Is your feature request related to a problem? Please describe.**
Add some badges to the top of readme.md file, since the badges will be colorful and more attractive at the top of README.md file than the text the users will like it and after linking the badges it will also help the users to navigate to different sections/ webpages like PRs, issues, contributors, etc.

image

### **Describe the solution you'd like**
I will add some badges at the top and also will add links related to them so that the README.md file will look more interactive and awesome.

### **Work under/as**
- [x] LGMSOC
- [ ] Contributor

### PR DESCRIPTION
**Pull request by Umesh-01**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/soumik2012/Chathub/blob/master/CONTRIBUTING.md)?

Yes

### Description

I have added some badges with the links related to them to the README.md file, after clicking on them the user will be redirected to the related section/web page.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.

## Related Issues or Pull Requests

This fixes #39

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,20 @@
...
+
+<a href="https://github.com/soumik2012/Chathub/graphs/contributors"><img src="https://img.shields.io/github/contributors/soumik2012/Chathub?color=brightgreen"></a>
+<a href="https://github.com/soumik2012/Chathub/stargazers"><img src="https://img.shields.io/github/stars/soumik2012/Chathub?color=0059b3"></a>
+<a href="https://github.com/soumik2012/Chathub/network/members"><img src="https://img.shields.io/github/forks/soumik2012/Chathub?color=yellow"></a>
+<a href="https://github.com/soumik2012/Chathub/issues"><img src="https://img.shields.io/github/issues/soumik2012/Chathub?color=0059b3"></a>
+<a href="https://github.com/soumik2012/Chathub/issues?q=is%3Aissue+is%3Aclosed"><img src="https://img.shields.io/github/issues-closed-raw/soumik2012/Chathub?color=yellow"></a>
+<a href="https://github.com/soumik2012/Chathub/pulls"><img src="https://img.shields.io/github/issues-pr/soumik2012/Chathub?color=brightgreen"></a>
+<a href="https://github.com/soumik2012/Chathub/pulls?q=is%3Apr+is%3Aclosed"><img src="https://img.shields.io/github/issues-pr-closed-raw/soumik2012/Chathub?color=0059b3"></a> 
+</div>
```
Comment (soumikbaithalu) [inline README.md:21]: Put batch under the image

File: README.md [hunk 2]
```diff
@@ -1,4 +1,20 @@
...
 
 ![Untitled design](https://user-images.githubusercontent.com/66599363/123587387-d251a680-d803-11eb-883c-63284340245b.jpg)
```
Comment (soumikbaithalu) [inline README.md:24]: Add a head line Overview

File: README.md (hunk 1 above)
Comment (soumikbaithalu) [reply README.md:21]: On 16 and 17 lines put one line gap between them

File: README.md (hunk 2 above)
Comment (soumikbaithalu) [reply README.md:24]: @Umesh-01

File: README.md (hunk 1 above)
Comment (Umesh-01) [reply README.md:21]: done 👍

File: README.md (hunk 2 above)
Comment (Umesh-01) [reply README.md:24]: @soumik2012 head line overview means??

Comment (Umesh-01) [reply README.md:24]: horizontal rule??
like this??

-----

Comment (soumikbaithalu) [reply README.md:24]: @Umesh-01 No just add a heading `OverView` like this  
```
# Overview 

This is a Messenger App, made with react, styled with the help of material-ui
and deployed with the help of firebase. If you are react Lover feel free to
```

**Review by soumikbaithalu — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Umesh-01) [reply README.md:24]: Okk, got it...

**Comment by Umesh-01** [comment]
@soumik2012 check it now... 🙂

**Review by soumikbaithalu — APPROVED** [review]
LGTM 💯

**Pull request merged**